lib/dataUtils.js

```diff
@@ -7,16 +7,18 @@ var assign = require('lodash/assign');
 
 var logMessage = function(debug, functionName, message, details) {
   if (debug) {
-    var finalMessage = message;
-
-    if (details && debug !== 'instrumentation')  {
-      if (Buffer.isBuffer(details) || typeof details === 'string') {
-        finalMessage = message + '\n' + details;
-      } else if (typeof details === 'object') {
-        finalMessage = message + '\n' + JSON.stringify(details);
+    try {
+      var finalMessage = message;
+      if (details && debug !== 'instrumentation')  {
+        if (Buffer.isBuffer(details) || typeof details === 'string') {
+          finalMessage = message + '\n' + details;
+        } else if (typeof details === 'object') {
+          finalMessage = message + '\n' + JSON.stringify(details);
+        }
       }
+      console.log('MOESIF: [' + functionName + '] ' + finalMessage);
```

Comment:
`debug` check should be here?

Reply:
that is our main console log inside logMessage